operation/fhir-operation-bulkdata/src/main/java/org/linuxforhealth/fhir/operation/bulkdata/client/BulkDataClient.java

```diff
@@ -548,12 +548,12 @@ private void addExecutionResponse(PollingLocationResponse pollingLocationRespons
 
         // Export Jobs with data in the exitStatus field
         if (!"COMPLETED".equals(exitStatus) && !"bulkimportchunkjob".equals(executionResponse.getJobName())) {
-            List<String> resourceTypeInfs = Arrays.asList(exitStatus.split("\\s*:\\s*"));
+            List<String> resourceTypeInfs = Arrays.asList(exitStatus.split(":"));
             List<PollingLocationResponse.Output> outputList = new ArrayList<>();
             for (String resourceTypeInf : resourceTypeInfs) {
                 String resourceType = resourceTypeInf.substring(0, resourceTypeInf.indexOf("["));
                 String[] resourceCounts =
-                        resourceTypeInf.substring(resourceTypeInf.indexOf("[") + 1, resourceTypeInf.indexOf("]")).split("\\s*,\\s*");
+                        resourceTypeInf.substring(resourceTypeInf.indexOf("[") + 1, resourceTypeInf.indexOf("]")).split(",");
```

Comment:
I'm going to assume here that there's no trimming required, right?

Reply:
yeah, i tried to cover that in the commit message:
> In BulkDataClient.addExecutionResponse we're parsing our own output...there's no reason to expect whitespace.

to test it, I did an export of multiple resource types and changed the config to ensure that we end up with multiple "objects" for at least one of those resource types